src/import/importd.c

```diff
@@ -632,6 +635,11 @@ static int manager_new(Manager **ret) {
         if (!m)
```

Comment:
Please replace `new0()` with `new()`, as now the allocated object is not necessary to be initialize with 0.

Reply:
Fixed, thanks.